api/docs/release.dox

```diff
@@ -232,6 +232,8 @@ Further non-compatibility-affecting changes include:
    insertion of instrumentation and instrumentation optimizations. Its primary purpose
    is to enable debugging of instrumentation sequences and detection of
    instrumentation-induced bugs.
```

Comment:
Would it be better to have used the original PR #4938?  Is that PR being split into any further pieces or is this the rest of it after the drmgr PR?  Then we would have our review history here, and presumably Github would give us a nice "view changes since last review' or at least we could select from which commits to view and not have to re-review a whole lot of code we've already reviewed...with this I feel like I have to go manually check out both of your branches and diff them myself to see what to look at.

Reply:
I would have used the original PR but there are two more PRs that will come out of #4938. One with the addition of an option in drcachesim to use drstatecmp and a drcachesim delay test, and another PR for the fuzz-app and the test that comes with it.

Reply:
In terms of diff, this PR has essentially a copy paste of the drstatecmp files from #4938 apart from changing the calls to the modified drmgr all-events registration function (changed in #5059 and thus post-#4938).

Reply:
OK, SGTM.